fido2-demo/demo/src/main/resources/application-local.yml

```diff
@@ -22,4 +24,8 @@ logging:
     com.linecorp.line: DEBUG
     org.springframework.web: DEBUG
     org.hibernate: DEBUG
+app:
+  origins:
+      - android:aaa-bbb
+      - ios:aaa-bbb
```

Comment:
I think there are only examples for native applications. How do you think if you add an example for web application?

Reply:
Since this configuration is only needed for the app, I've added a note through comments for the web case.